### PR TITLE
Hotfix/index file for browserify

### PR DESCRIPTION
**Pull request by mandeepm91**
added `index.js` file to make it work with webpack/browserify

### CODE REVIEW

**Comment by CrackerakiUA** [comment]
I am not using it, but I am glad that it will help you :)

File: index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+require('./source/js/init.js');
+module.exports = 'ngImgCrop';
```
Comment (rickerd) [inline index.js:2]: Or is the name ngImgCropFullExtended?
The name has changed since the original one doesn't update anymore.

Comment (CrackerakiUA) [reply index.js:2]: @mandeepm91 export effect the code?

Comment (mandeepm91) [reply index.js:2]: @CrackerakiUA Not sure I understood your question. I was able to bundle using browserify after adding this, if that's what you're asking ? I picked this up from an old issue where someone had suggested this change but did not create a PR

Comment (CrackerakiUA) [reply index.js:2]: @rickerd generally if it's working on their project, then it's not a problem.

**Comment by rickerd** [comment]
True

**Pull request merged**